recipes/scvi/meta.yaml

```diff
@@ -1,47 +1,29 @@
 {% set name = "scvi" %}
-{% set version = "0.4.1" %}
+{% set version = "0.5.0" %}
 
 package:
   name: '{{ name|lower }}'
   version: '{{ version }}'
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 23cab6850f433e85aa23c12ddfe7fb74e12b481d584e5f519e5f51f60d6e64cf
+  sha256: ea56364ce84c2dd601735d465aea04cefdcb709311bb3892cd5cab02d1c151a2
 
 build:
   noarch: python
-  number: 1
+  number: 0
   script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
   host:
+    - python >=3.7
+    - pip
+
     - pytest-runner >=4.0
-    - python >=3.6
+    - python >=3.7
     - pip >=19.0
-    - matplotlib >=3.0
-    - scikit-learn >=0.20
-    - scipy >=1.2
-    - h5py >=2.9
-    - pandas >=0.24
-    - loompy >=2.0.17
-    - tqdm >=4
-    - anndata >=0.6
-    - xlrd >=1.2
-    - jupyter >=1.0.0
-    - nbconvert >=5.5.0
-    - nbformat >=4.4.0
-    - ipython >=7.5
-    - umap-learn >=0.3.7
-    - seaborn >=0.9.0
-    - leidenalg >=0.7.0
-    - python-igraph >=0.7.1
-    - hyperopt >=0.1.2
-    - mongodb >=4
-    - scanpy >=1.4
-    - colour >=0.1
+
   run:
-    - python >=3.6
```

Comment:
It's best to keep an explicit python entry here and it should match what's in the `host:` section.